### PR TITLE
Feat/about page

### LINKED ISSUE #11: Fix About us page
Fix about us as the design sketches.

### PR DESCRIPTION
**Pull request by emilps**
Added design and own component for block with text and image.

closes #11

### CODE REVIEW

File: src/components/Content.js [hunk 1]
```diff
@@ -1,19 +1,90 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React from 'react';
+import PropTypes from 'prop-types';
+import './all.sass';
 
-export const HTMLContent = ({ content, className }) => (
-  <div className={className} dangerouslySetInnerHTML={{ __html: content }} />
-)
+export const HTMLContent = ({ content, className }) => {
```
Comment (eirikdahlen) [inline src/components/Content.js]: What is all this for?

File: src/cms/cms.js [hunk 2]
```diff
@@ -32,4 +35,58 @@ CMS.registerPreviewStyle('./../components/all.sass');
 
 CMS.registerWidget('relatedProduct', controlComponent);
 
+CMS.registerEditorComponent({
```
Comment (eirikdahlen) [inline src/cms/cms.js]: What is this for?

Comment (emilps) [reply src/cms/cms.js]: It's a component for the markdown that allows the user to create a block with image and text.

File: src/components/Content.js (hunk 1 above)
Comment (emilps) [reply src/components/Content.js]: It's for rendering the imageblock in the markdown. It finds the object and creates a div with text and image

**Pull request merged**